beets/autotag/hooks.py

```diff
@@ -78,7 +78,8 @@ def __init__(self, tracks, album=None, album_id=None, artist=None,
                  original_year=None, original_month=None,
                  original_day=None, data_source=None, data_url=None,
                  discogs_albumid=None, discogs_labelid=None,
-                 discogs_artistid=None, **kwargs):
+                 discogs_artistid=None, vgmdb_album_id=None,
+                 vgmdb_artist_id=None, vgmdb_game=None, **kwargs):
```

Comment:
There should be no need to add any new fields here (nor to touch `hooks.py` at all); the new data can just go in `kwargs`.